cmd/jujud/updateseries/updateseries.go

```diff
@@ -0,0 +1,396 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package updateseries
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"github.com/juju/loggo"
+	"github.com/juju/utils/arch"
+	"github.com/juju/utils/fs"
+	jujuos "github.com/juju/utils/os"
+	"github.com/juju/utils/series"
+	"github.com/juju/utils/shell"
+	"github.com/juju/utils/symlink"
+	"github.com/juju/version"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/juju/agent/tools"
+	"github.com/juju/juju/cmd/jujud/agent"
+	cmdutil "github.com/juju/juju/cmd/jujud/util"
+	"github.com/juju/juju/juju/paths"
+	"github.com/juju/juju/mongo"
+	"github.com/juju/juju/service"
+	"github.com/juju/juju/service/common"
+	"github.com/juju/juju/service/systemd"
+)
+
+var logger = loggo.GetLogger("juju.cmd.jujud.updateseries")
+
+const updateSeriesCommandDoc = `
+Update Juju agents running on this machine to start with Systemd.
```

Comment:
this is trivial, but systemd should be written as systemd, all together, lowercase. That's the naming convention for system daemons on linux.

Also, in theory wouldn't this be expanded as needed for later upgrades if any? I'm curious if this would be invoked or needed on a xenial to bionic (systemd to systemd) upgrade.

Reply:
This will need to be expanded for the xenial to bionic upgrade, a phase 2.  Non LTS upgrades from a systemd using series to another, should copy the tools as well.  Adding that.

The command description needs some wordsmithing, suggestions are accepted.